src/SpendPermissionManager.sol

```diff
@@ -500,12 +531,17 @@ contract SpendPermissionManager is EIP712 {
         bytes32 hash = getHash(spendPermission);
         _isApproved[hash] = true;
         emit SpendPermissionApproved(hash, spendPermission);
+        return true;
     }
 
     /// @notice Revoke a spend permission.
     ///
+    /// @dev Emits a {SpendPermissionRevoked} event if the spend permission is newly revoked.
+    ///
     /// @param spendPermission Details of the spend permission.
     function _revoke(SpendPermission memory spendPermission) internal {
```

Comment:
What's the intuition for not having `_revoke` have a `returns (bool)`? Even though it will never return false, it could be cleaner reading to have it for parity?